classes/class-wc-connect-help-view.php

```diff
@@ -156,29 +156,45 @@ protected function get_health_items() {
 			// Check that we are able to talk to the WooCommerce Services server
 			$schemas = $this->service_schemas_store->get_service_schemas();
 			$last_fetch_timestamp = $this->service_schemas_store->get_last_fetch_timestamp();
+			$refresh_link = sprintf(
+				wp_kses(
+					__( '<a href="%s">Refresh</a>', 'woocommerce-services' ),
+					array( 'a' => array( 'href' => array() ) )
+				),
+				add_query_arg(
+					array(
+						'page' => 'wc-status',
+						'tab' => 'connect',
+						'refresh' => 'true',
+					),
+					admin_url( 'admin.php' )
+				)
+			);
 			if ( ! is_null( $last_fetch_timestamp ) ) {
 				$last_fetch_timestamp_formatted = sprintf(
-					_x( 'Last updated %s ago', '%s = human-readable time difference', 'woocommerce-services' ),
-					human_time_diff( $last_fetch_timestamp )
+					_x( 'Last updated %s ago. %s', '1st %s = human-readable time difference, 2nd %s = refresh url', 'woocommerce-services' ),
```

Comment:
Multiple tokens require the argument position to be specified. `%1$s` and `%2$s`.